src/visual.ts

```diff
@@ -877,11 +877,12 @@ module powerbi.extensibility.visual {
 
             const category: DataViewCategoryColumn = dataView.categorical.categories[0];
             let categoryExpr: any = category.source && category.source.expr ? category.source.expr as any : null;
-
-            // category.source.queryName contains wrong table name if table was renamed! category.source.expr.source.entity contains correct table name.
+            let filteringColumn: any = categoryExpr && categoryExpr.arg && categoryExpr.arg.arg && categoryExpr.arg.arg.property ? categoryExpr.arg.arg.property : category.source.displayName;
```

Comment:
Why is it `any`? I suppose that it should be `string` instead

Reply:
Oh, yeah, in that case it is better. Thanks